js/ui/lookup.js

```diff
@@ -963,6 +994,10 @@ const Lookup = DropDownList.inherit({
                 break;
             case 'cleanSearchOnOpening':
             case '_scrollToSelectedItemEnabled':
```

Comment:
It seems that we already have the option that enabled an item centering  in material theme. Can we use only one option for that purposes.